std/file.d

```diff
@@ -2304,6 +2308,39 @@ if (isConvertibleToString!R)
     }
 }
 
+version (Windows)
```

Comment:
Windows-only symbol documentation needs to be in a `version(StdDdoc)`, otherwise it won't be visible on dlang.org.